### DISCUSSION TITLE
Bookmarks

### DISCUSSION THREAD

**Discussion by TheFlipside** [45Drives/cockpit-navigator#24] [Ideas]
Hey, it would be helpful to add a functionality to set bookmarks for directories and files.

This way it would make it easier for example in case a certain configuration file which gets modified regularly, e.g. a whitelist for a proxy server, to navigate to the file quicker than having to go through all the sub directories manually.

**Comment by joshuaboud** [comment]
Hi, bookmarks are a good idea but may not be implemented any time soon. In the mean time though, do note that you can start typing the path to the file in the address bar and it will show an autocomplete dropdown list to make navigation to specific files a little faster

**Comment by tHe-BiNk** [comment]
I use Webmin with bookmarks in the file-manager on the main server. Bookmarks would be very helpful in the cockpit navigator. It would save me a lot of time. TIA!

Thank you for this great add-on to cockpit!
